conf/xanadu.config

```diff
@@ -0,0 +1,38 @@
+params {
+    config_profile_description = 'The UConn HPC profile'
+    config_profile_contact     = 'noah.reid@uconn.edu'
+    config_profile_url         = 'https://bioinformatics.uconn.edu/'
+
+    // max resources
+    max_memory                 = 2.TB
+    max_cpus                   = 64
+    max_time                   = 21.d
+
+    // Path to shared singularity images
+    singularity_cache_dir      = '/isg/shared/databases/nfx_singularity_cache'
+
+}
+
+process {
+    executor                   = 'slurm'
+    queue                      = { task.memory <= 245.GB ? 'general' : ( task.memory <= 512.GB ? 'himem' : 'himem2' ) }
+    
+    clusterOptions             = { [    
+        task.memory <= 245.GB ? '--qos=general' : '--qos=himem',
+        // provide hardware constraints for particular processes
+        //"${task.process.tokenize(':')[-1]}" ==~ /[BWAbwa]{3}[-_][MEme]{3}2.*/ ? '--constraint="AVX|AVX2|AVX512|SSE41|SSE42"' : ''
+    ].join(' ').trim() }

```

Comment:
I'm guessing you don't need the `].join(' ').trim() }` part if you keep the L22 and L23 as comments, but I'm impressed